src/components/Profile/EditUserForm.js

```diff
@@ -104,19 +99,13 @@ export default function EditUserForm() {
             {...layout}
             name="basic"
             onFinish={editUserSubmit}
-            initialValues={{
```

Comment:
@simonesquad I'd love your input on whether you think this is a breaking change. All seemed alright to me after deleting this but just want to confirm.